conf2sql.pl

```diff
@@ -29,16 +29,23 @@
 
 die "$program: missing --dbdriver option\n" unless defined($opts{'dbdriver'});
 die "$program: missing --dbname option\n" unless defined($opts{'dbname'});
+if (!$opts{'dbdriver'} =~ /sqlite/i) {
```

Comment:
sqlite note user server/pass/user like mysql and pgsql, is to differentiate, and i use dbname as sqlite dsn.